src/collective/importexport/browser/import_view.py

```diff
@@ -258,16 +274,61 @@ def export_file(result, header_mapping, request=None):
                     continue
                 if value is field.missing_value:
                     continue
+                if IRichTextValue.providedBy(value):
+                    # Convert to plain text
+                    transforms = api.portal.get_tool('portal_transforms')
+                    value = transforms.convertTo(
+                                target_mimetype='text/plain',
+                                orig=value.raw_encoded,
+                                encoding=value.encoding,
+                                mimetype=value.mimeType)
+                    value = safe_unicode(value.getData())
+                    break
+
+                if IList.providedBy(field):
+                    # Check if items in list point to objects and use titles
+                    # instead
+                    names=[]
+                    for v in value:
+                        # Loop through choices
+                        query = dict(id=v)
+                        catalog = api.portal.get_tool("portal_catalog")
+                        results = catalog(**query)
+                        if results:
+                            names.append(results[0].Title)
+                    if names:
+                        value = names
+                    value = ', '.join(value)
+                    break
+
+                if INamed.providedBy(value):
+                    # Image, file field
+                    value = u'{0}/@@download/{1}'.format(row.getURL(),
+                        fieldid)
+                    break
+
                 serializer = ISerializer(field)
                 value = serializer(value, {})
                 break
-            items.append(value)
 
-#        log.debug(items)
-        writer.writerow(items)
-    csv_attachment = csv_file.getvalue()
-    csv_file.close()
-    return csv_attachment
+            # Added due to an ascii error https://github.com/collective/collective.importexport/issues/1
```

Comment:
You should reference the ticket number in the commit instead, so it would be linked with the ticket, instead of adding a comment here.